modules/cluster-logging-deploy-clo.adoc

```diff
@@ -7,39 +7,45 @@
 
 You can use the {product-title} web console to install the Cluster Logging Operator.  
 
+[NOTE]
+====
+You cannot create a Project starting with `openshift-` using the web console or 
+by using the `oc new-project` command. You must create Namespace using a YAML 
+object file and run the `oc create -f <file-name>.yaml` command, as shown.
```

Comment:
s/file-name/file_name/

Reply:
@adellape Are we supposed to use `file_name`? I see far more `file-name` instances in the repo.

Reply:
Our replaceable examples here only show use of underscores: https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference

Though actually, IBM SG says to use "filename" (one word) when a variable, anyway. /shruggie

Reply:
Also IBM:

> If you must use a multiple-word variable name, remove the spaces between the words, or connect the
words with underscores.

Reply:
Wonder if it would be worth a find/replace across the docs. Plenty of `file-name` references in there. 
I can change these in another PR.